src/utils/helper_embedding.py

```diff
@@ -16,3 +18,84 @@ def reshape_df(df: pd.DataFrame, col_name: str) -> pd.DataFrame:
         return df[['word', col_name]]
     except Exception as error:
         print(f"Cause: {error}")
+
+
+def calculate_cosine_similarity(a: list, b: list) -> float:
+    """
+    Computes the cosine-similarity of a word with each word in a dataframe.
```

Comment:
excellent docstrings! I dont do it as much as I should, it's really good practice so nice one!